cql3/Cql.g

```diff
@@ -1232,6 +1233,26 @@ dropRoleStatement returns [std::unique_ptr<drop_role_statement> stmt]
       { $stmt = std::make_unique<drop_role_statement>(name, if_exists); }
     ;
 
+named_parameter[std::unordered_map<sstring, sstring> &params]
+    : k1=cident '=>' v1=constant
+    {
+        params.emplace(k1->to_string(), v1->get_raw_text());
```

Comment:
get_raw_text loses the type infomation, so 0x616263 becomes "0x616263" instead of a blob ABC.

Reply:
For now the `CALL` statement has no concept of types on the callers side - it's taken in as a string and converted if necessary inside the appropriate routine. This should be enhanced further down the road ofc, but for now should satisfy a working mvp for the call statements